vm/interpreter/src/gas_block_store.rs

```diff
@@ -145,43 +145,40 @@ where
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::price_list_by_epoch;
     use cid::Code::Blake2b256;
     use db::MemoryDB;
+    use fvm::gas::price_list_by_network_version;
     use fvm::gas::Gas;
     use networks::{ChainConfig, Height};
 
     #[test]
     fn gas_blockstore() {
         let calico_height = ChainConfig::default().epoch(Height::Calico);
+        let network_version = ChainConfig::default().network_version(calico_height);
         let db = MemoryDB::default();
         let gbs = GasBlockStore {
-            price_list: PriceList {
-                ipld_get_base: 4,
-                ipld_put_base: 2,
-                ipld_put_per_byte: 1,
-                ..price_list_by_epoch(0, calico_height)
-            },
-            gas: Rc::new(RefCell::new(GasTracker::new(Gas::new(5000), Gas::new(0)))),
+            price_list: price_list_by_network_version(network_version).clone(),
+            gas: Rc::new(RefCell::new(GasTracker::new(
+                Gas::new(5000000000000),
+                Gas::new(0),
+            ))),
             store: &db,
         };
         assert_eq!(gbs.gas.borrow().gas_used(), Gas::new(0));
         assert_eq!(to_vec(&200u8).unwrap().len(), 2);
         let c = gbs.put(&200u8, Blake2b256).unwrap();
-        assert_eq!(gbs.gas.borrow().gas_used(), Gas::new(2002));
+        assert_eq!(gbs.gas.borrow().gas_used(), Gas::new(356242));
```

Comment:
Try computing `356242` from the price list.

Reply:
Done